packages/rich-text-react-renderer/src/__test__/documents/ul.ts

```diff
@@ -0,0 +1,62 @@
+import { Document } from '@contentful/rich-text-types';
+
+export default {
+  data: {},
+  content: [
```

Comment:
It seems that we need to find a better way of organising mocks. I will take of it once we merge this PR.